src/components/permissions/routes.ts

```diff
@@ -198,7 +198,7 @@ const routeVerification = async (request: any, reply: any) => {
 // Serve prefs to front-end
 const routeGetPrefs = async (request: any, reply: any) => {
   const prefs = JSON.parse(
-    readFileSync(path.join(getAppEntryPointDir(), '../preferences.json'), 'utf8')
+    readFileSync(path.join(getAppEntryPointDir(), '../preferences/preferences.json'), 'utf8')
```

Comment:
Note that the prefs file is now inside a `preferences` folder -- this is the easiest way to put the prefs in a docker volume.